src/client/CodeEditor/getOrCreateEditor.ts

```diff
@@ -41,17 +41,38 @@ import { ThemeLabel, themeOptionsByLabel } from '../themes';
 import { keymap } from '@codemirror/view';
 import { basicSetup } from './basicSetup';
 import { InteractRule } from '@replit/codemirror-interact';
-import rainbowBrackets from '../CodeEditor/rainbowBrackets';
+import rainbowBrackets from './rainbowBrackets';
 import { cssLanguage } from '@codemirror/lang-css';
 import { javascriptLanguage } from '@codemirror/lang-javascript';
 import { copilot } from './Copilot';
+import { type WorkerShape } from '@valtown/codemirror-ts/src/worker';
+import * as Comlink from 'comlink';
+import {
+  tsAutocompleteWorker,
+  tsFacetWorker,
+  tsHoverWorker,
+  tsLinterWorker,
+  tsSyncWorker,
+} from '@valtown/codemirror-ts';
 
 const DEBUG = false;
 
 // Enables TypeScript +JSX support in CodeMirror.
 const tsx = () =>
   javascript({ jsx: true, typescript: true });
 
+const innerWorker = new Worker(
```

Comment:
This will likely fail in an SSR context, which is there in the VizHub app but not in the VZCode demo app (so it works fine here). We can circle back and fix this later.